crates/iceberg/src/spec/manifest.rs

```diff
@@ -781,10 +805,13 @@ impl ManifestMetadata {
     }
 }
 
+/// Reference to [`ManifestEntry`].
+pub type ManifestEntryRef = Arc<ManifestEntry>;
```

Comment:
Sorry still new to Rust but is suffixing with `Ref` a common naming convention for reference counted variables?

Reply:
It's hard to say if it's common in all rust projects, but some projects I've worked on use this convention.